server/utils/licenceFactory.js

```diff
@@ -1,4 +1,5 @@
 const {conditionsOrder} = require('../models/conditions');
+const NO_INPUT_MESSAGE = 'INPUT REQUIRED';
```

Comment:
Short term fix to enable us to display that something is missing on the next page. Will be updated when validation display is decided